swap/prices.go

```diff
@@ -33,9 +33,15 @@ allowing for a multi-currency design.
 */
 
 // Placeholder prices
+// Based on a very crude calculation: average monthly cost for bandwidth in the US / average monthly bill for bandwidth in the US
```

Comment:
I think we should use an average cloud storage price instead of internet provider cost

Reply:
Based on the calculation in the chat, I leave it as-is. This is not because I don't think you are right, but because I think that both presented options have flaws which we must consider. Changing it now to a different price will possibly bring up other issues and for the workshop the chosen price is sufficient. 

How would you go about defining this in a more proper manner? In my opinion, we must first agree on a methodology on how to get the price, after which we can apply this methodology. Shall we plan a workshop in Japan with the incentives team and research?